packages/design-system/src/atoms/qrcode/index.tsx

```diff
@@ -0,0 +1,25 @@
+import QRCode from 'react-qr-code'
+import { View, StyleSheet } from 'react-native'
+
+export type QRCodeProps = {
+  value: string
+  size?: number
+}
+
+const useStyles = () => {
+  const styles = StyleSheet.create({
+    container: {
+      justifyContent: 'center',
+    },
+  })
+  return styles
+}
```

Comment:
We don't need styles to be in hook. They can be declared globally 

```suggestion
  const styles = StyleSheet.create({
    container: {
      justifyContent: 'center',
    },
  })
  return styles
```

Reply:
Do we only need useStyles hook when we work with useMemo?